packages/components/psammead-synopsis/src/index.jsx

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
+import { shape, string } from 'prop-types';
+import { C_METAL } from '@bbc/psammead-styles/colours';
+import { GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import { getSansBold } from '@bbc/psammead-styles/font-styles';
+import { getDoublePica } from '@bbc/gel-foundations/typography';
+
+const Synopsis = styled.p`
+  ${({ script }) => script && getDoublePica(script)};
+  ${({ service }) => getSansBold(service)}
+  color: ${C_METAL};
+  padding-bottom: 2 * ${GEL_SPACING_DBL};
```

Comment:
You can import and use `GEL_SPACING_QUAD` instead. 
You should likely reset paragraph padding too on the other SIDES, to override default paragraph spacing. e.g.
`padding: 0 0 ${GEL_SPACING_QUAD} 0;`